CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.5.0
+
+**Breaking changes**
+
+* Remove the semi colon in golden file names to support using the package on a Windows machine. 
+  
+  This is a breaking change for users who have generated golden files with the previous version of the library. The golden file names will now be `preview/${windowConfig.name}-${name.snakeCase}$localSuffix.png` instead of `preview/${windowConfig.name}:${name.snakeCase}$localSuffix.png`.
+
+  To resolve this, you can either rename the golden files manually or regenerate them.
```

Comment:
You can propose a script to help with the migration:
```bash
#!/bin/bash

# Function to rename files in directories named "preview"
rename_files_in_preview() {
    # Find directories named "preview"
    find . -type d -name "preview" | while read -r dir; do
        echo "Processing directory: $dir"
        # Find files within these directories
        find "$dir" -type f | while read -r file; do
            # New filename by replacing ':' with '-'
            new_name=$(echo "$file" | sed 's/:/-/g')
            if [ "$file" != "$new_name" ]; then
                mv "$file" "$new_name"
                echo "Renamed $file to $new_name"
            fi
        done
    done
}

# Call the function
rename_files_in_preview
```

Reply:
Done!